src/sentry/discover/compare_timeseries.py

```diff
@@ -69,6 +84,25 @@ def make_rpc_request(
         sampling_mode=None,
     )
 
+    assert snuba_params.start is not None
+    assert snuba_params.end is not None
+
+    with sentry_sdk.isolation_scope() as scope:
+        path, query = format_api_call(
+            organization.slug,
+            query=query_parts["query"],
+            useRpc=1,
+            project=snuba_params.project_ids[0],
+            yAxis=query_parts["selected_columns"][0],
+            dataset="spans",
+            interval=snuba_params.granularity_secs,
+            start=snuba_params.start.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
+            end=snuba_params.end.strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
+            sampling="BEST_EFFORT",
```

Comment:
Should this query be running with the BEST_EFFORT sampling mode? Do the alert subscriptions also use sampled storage?